spec/unit/inventory_spec.rb

```diff
@@ -51,4 +51,52 @@
       expect(resource.children_by_type('type-03', true).size).to eq(1)
     end
   end
+
+  describe '#ResultFetcher' do
+    it 'should fetch two first pages' do
```

Comment:
For consistency, those tests should be in the present tense of the third person, as pointed here: http://www.betterspecs.org/#should

```ruby
it 'fetches first two pages'
```